FastSurferCNN/utils/common.py

```diff
@@ -178,31 +180,30 @@ def pipeline(
     *,
     pipeline_size: int = 1,
 ) -> Iterator[Tuple[_Ti, _T]]:
-    """Pipeline a function to be executed in the pool.
+    """
+    Pipeline a function to be executed in the pool.
 
     Analogous to iterate, but run func in a different
     thread for the next element while the current element is returned.
 
-    Parameters [MISSING]
+    Parameters
     ----------
     pool : Executor
+        Thread pool executor for parallel execution.
 
     func : Callable[[_Ti], _T] :
-        function to use
+        Function to use.
 
     iterable : Iterable[_Ti]
-
-    * :
-        [MISSING]
+        Iterable containing input elements.
 
     pipeline_size : int
-        size of the pipeline
-        (Default value = 1)
+        Size of the processing pipeline (default is 1).
 
     Returns
     -------
-        [MISSING]
-
+    Iterator[Tuple[_Ti, _T]]
+        Iterator yielding input elements and corresponding results.
```

Comment:
... yielding elements from `iterable` and corresponding results from `func(element)`